package.json

```diff
@@ -199,8 +199,12 @@
     "export:copy": "cpx \"dist/mozilla-central/{addon/**,bootstrap.js,common/**,data/**,experiments.json,install.rdf.in,moz.build,package.json}\" ../mozilla-central/browser/extensions/activity-stream",
     "export:copytests": "if [ \"$SYMLINK_TESTS\" != \"true\" ]; then cpx \"test/functional/mochitest/**\" ../mozilla-central/browser/extensions/activity-stream/test/functional/mochitest; fi",
     "export:linktests": "if [ \"$SYMLINK_TESTS\" = \"true\" ]; then ln -s ../../../../activity-stream/test ../mozilla-central/browser/extensions/activity-stream/test; fi",
-    "export:patch": "(cd ../mozilla-central && patch -Np1 --no-backup-if-mismatch -r rejected-export.diff; rm -f rejected-export.diff) < mozilla-central/moz.build.patch",
-    "export:patch2": "(cd ../mozilla-central && patch -Np1 --no-backup-if-mismatch -r rejected-export.diff; rm -f rejected-export.diff) < mozilla-central/prefs_general.diff",
+    "export:patch": "(cd ../mozilla-central && patch -Np1 --no-backup-if-mismatch -r rejected-export.diff; rm -f rejected-export.diff) < mozilla-central-patches/moz.build.patch",
+    "export:patch2": "(cd ../mozilla-central && patch -Np1 --no-backup-if-mismatch -r rejected-export.diff; rm -f rejected-export.diff) < mozilla-central-patches/prefs_general.diff",
+    "buildmc": "npm-run-all buildmc:*",
+    "prebuildmc": "npm run export:clean && npm run  export:patch",
+    "buildmc:copy": "cpx \"system-addon/**/*\" ../mozilla-central/browser/extensions/activity-stream",
+    "startmc": "npm run prebuildmc && npm run buildmc:copy -- -w",
```

Comment:
My suspicion is that over time, we're going to want to have an in-github build system and an export system, like we do today, and a bunch of these buildmc pieces will really be part of the export.  I don't think there's anything worth changing now, but it's worth keeping in mind.